CONTRIBUTORS.txt

```diff
@@ -1,48 +1,43 @@
+Maintainers
+-----------
+- Claudiu Popa <pcmanticore@gmail.com>
+- Pierre Sassoulas <pierre.sassoulas@gmail.com>
+- Daniël van Noord <13665637+DanielNoord@users.noreply.github.com>
+- Marc Mueller <30130371+cdce8p@users.noreply.github.com>
+- Hippo91 <guillaume.peillex@gmail.com>
+- Łukasz Rogalski <rogalski.91@gmail.com>
+- Ashley Whetter <ashley@awhetter.co.uk>
+- Bryce Guinta <bryce.paul.guinta@gmail.com>
+- Jacob Walls <jacobtylerwalls@gmail.com>
+- Dimitri Prybysh <dmand@yandex.ru>
+  - multiple-imports, not-iterable, not-a-mapping, various patches.
```

Comment:
Just in terms of parse ability, do want this to be a `*` or something?

Reply:
I'm wondering is we should not instead add a link to : https://github.com/PyCQA/pylint/pulls/areveny

We'd need a "database" mail => github handles though. But this would be highly automatable and  up to date...

Reply:
I think it should be possible to get that info based on GitHub's rest API. But that's really something for the future, as in, there are more pressing matters.

Reply:
Please review commit by commits the new global diff will probably be horrendous. I hesitated to create another PR but let's keep it simple 😄

Reply:
[4f25ef0](https://github.com/PyCQA/pylint/pull/5909/commits/4f25ef010cfa7d08ae15e7c15e577eb222bf8db2)